tests/unit/src/kakarot/instructions/test_environmental_information.cairo

```diff
@@ -140,3 +140,39 @@ func test__exec_extcodecopy__should_handle_address_with_no_code{
 
     return ();
 }
+
+@view
+func test__returndatacopy{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}() {
+    // Given
+    alloc_locals;
+
+    let (bytecode) = alloc();
+    let return_data: felt* = alloc();
```

Comment:
Is there a reason why you are using the (bytecode) syntax vs the return_data: felt* syntax? If there are no differences we should seek consistency